openghg/retrieve/icos/_retrieve.py

```diff
@@ -282,7 +282,7 @@ def _retrieve_remote(
     # Now extract the PIDs along with some data about them
     dobj_urls = filtered_sources["dobj"].tolist()
 
-    site_metadata = load_json("icos_atmos_site_metadata.json", internal_data=True)
+    site_metadata = load_json(filename="icos_atmos_site_metadata.json")
```

Comment:
Does this still run with the `internal_data` flag removed? This was to flag the data was within our openghg/data folder and so was needed for e.g. "icos_atmos_site_metadata.json" as this isn't a full path to the data file?

Reply:
Yeah I think we both made changes to that function at a similar time. I added the `path` kwarg so you could load any file, and you added the `internal_data` flag. Maybe we could more easily differentiate between them by creating a `load_internal_json` function and another `load_json` that just takes a full filepath? Quite a few bugs were introduced when I merged in both our changes.

Reply:
Yes we could approach this by using separate functions - that way it's still clear where the data is coming from (and what is internal to openghg).

Reply:
Copied over discussion to PR #537 where these original changes were made.